ui/app/components/dashboard/learn-more-card.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+/**
+ * @module DashboardLearnMoreCard
+ * DashboardLearnMoreCard component are used to display external links
+ *
+ * @example
+ * ```js
+ * <DashboardLearnMoreCard  />
+ * ```
+ */
+
+export default class DashboardLearnMoreCard extends Component {
+  get learnMoreLinks() {
+    return [
+      {
```

Comment:
Ah, interesting. Okay, so we have a `DocLinks` component which makes it so you don't have to use the full URL. Example below. However, I see you're using a HDS link component. Any chance HDS accounts for doc links? It can't just be a thing we need. If they don't, maybe make a comment here about this or in the documentation above, and put a request in to HDS to account for it?

```
 <DocLink @path="/vault/tutorials/auth-methods/multi-factor-authentication">
      Learn more
    </DocLink>
```

Reply:
Good to know! Looking at [HDS documentation](https://helios.hashicorp.design/components/link/standalone) it doesn't look like it takes into account DocLinks - it dynamically lets you pass in a `@route` or `@href`.